addons/mail/static/src/components/composer_suggested_partner/composer_suggested_partner.js

```diff
@@ -0,0 +1,41 @@
+odoo.define('mail/static/src/components/composer_suggested_partner/composer_suggested_partner.js', function (require) {
+'use strict';
+const { Component } = owl;
+const useStore = require('mail/static/src/component_hooks/use_store/use_store.js');
+// const { useState } = owl.hooks;
+
+class ComposerSuggestedPartner extends Component {
+
+    constructor(...args) {
+        super(...args);
+        // this.state = useState({});
+
+        useStore(props => {
+            const suggestedPartner = this.env.models['mail.suggested_partner'].get(props.suggestedPartnerLocalId);
+            return {
+                suggestedPartner: suggestedPartner ? suggestedPartner.__state : undefined,
+            };
```

Comment:
Model should be named `mail.suggested_recipient_info`, and `partner` own fields are not observed if `partner.__state` is not observed.

(btw I prefer "recipient" over "partner" just because "recipient" is more contextual)